smother/control.py

```diff
@@ -205,3 +240,14 @@ def iter_records(self, semantic=False, sort=True):
                     test_contexts = sorted(test_contexts)
                 for test_context in test_contexts:
                     yield src_context, test_context
+
+    def _normalize_combine_path(self, path):
+        aliases = None
+        if self.coverage and self.coverage.config.paths:
+            aliases = PathAliases()
```

Comment:
Let's only build PathAliases once within `__ior__`, instead of every time we call this function